kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/jaxb/JaxbMarshaller.java

```diff
@@ -277,26 +282,38 @@
         };
     }
 
-    private final JAXBContext jaxbContext;
+    protected JAXBContext jaxbContext;
 
-    private ClassLoader classLoader;
+    protected ClassLoader classLoader;
 
     public JaxbMarshaller(Set<Class<?>> classes, ClassLoader classLoader) {
         this.classLoader = classLoader;
+
+        buildMarshaller(classes, classLoader);
+    }
+
+    protected void buildMarshaller( Set<Class<?>> classes, final ClassLoader classLoader ) {
+
         try {
+            logger.debug("Additional classes for JAXB context are {}", classes);
             Set<Class<?>> allClasses = new HashSet<Class<?>>();
 
             allClasses.addAll(Arrays.asList(KIE_SERVER_JAXB_CLASSES));
             if (classes != null) {
                 allClasses.addAll(classes);
             }
-
+            logger.debug("All classes for JAXB context are {}", allClasses);
             this.jaxbContext = JAXBContext.newInstance( allClasses.toArray(new Class[allClasses.size()]) );
         } catch ( JAXBException e ) {
+            logger.error("Error while creating JAXB Marshaller due to {}", e.getMessage(), e);
```

Comment:
this will cause the same error to be logged twice: once here with this new log message and once by the default exception handler which should have the obligation to log all errors.